Userland/Games/Solitaire/Game.cpp

```diff
@@ -339,12 +339,13 @@ void Game::move_card(CardStack& from, CardStack& to)
     auto card = from.pop();
 
     card->set_moving(true);
-    m_focused_cards.clear();
     m_focused_cards.append(card);
```

Comment:
I kinda think we can get rid of all three of these lines:
```c++
    card->set_moving(true);
    m_focused_cards.clear();
    m_focused_cards.append(card);
```

`move_card` is only called when moving a card to a foundation stack. Semantically, when a card is moved to the foundation this way, I would say it is not "focused" - I *think* focused cards are meant to be the card(s) actively being dragged, and its purpose is to make sure dragged cards are painted on top of everything else.

Removing all three of these lines seems to work fine on my end, could you try that as well?

(As an aside, I actually wouldn't be opposed to `move_cards` becoming a lambda in the new `attempt_to_move_card_to_foundations` to ensure its only used for moving cards to a foundation)

Reply:
Thanks for the feedback! Sure, I'll give that a go. I wasn't clear about what focused meant - I might change its name if I can identify it for sure.